### PR TITLE
[WIP] additional fields for VIP, VirtualServer, Database, Cloud project

### PR DESCRIPTION
**Pull request by mkurek**
Connected to #1654

### CODE REVIEW

File: src/ralph/data_center/models/virtual.py [hunk 1]
```diff
@@ -1,26 +1,156 @@
...
+        return self.virtuals.count()
 
 
 class VirtualServer(BaseObject):
+    hostname = models.CharField(
+        verbose_name=_('hostname'),
+        blank=True,
+        default=None,
+        max_length=255,
+        null=True
```
Comment (szok) [inline src/ralph/data_center/models/virtual.py]: missing comma

File: src/ralph/data_center/models/virtual.py [hunk 2]
```diff
@@ -1,26 +1,156 @@
...
         verbose_name = _('database')
         verbose_name_plural = _('databases')
 
 
-class VIP(BaseObject):
+# =============================================================================
+# VIPs
+# =============================================================================
+class VIPType(NamedMixin):
+    class Meta:
```
Comment (ar4s) [inline src/ralph/data_center/models/virtual.py]: ditto

File: src/ralph/data_center/models/virtual.py [hunk 3]
```diff
@@ -1,26 +1,156 @@
...
+# =============================================================================
+class VirtualServerModel(NamedMixin):
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.PROTECT,
+        blank=True,
+        null=True,
+    )
+
+    class Meta:
```
Comment (ar4s) [inline src/ralph/data_center/models/virtual.py]: ditto

File: src/ralph/data_center/models/virtual.py [hunk 4]
```diff
@@ -1,26 +1,156 @@
...
 
 
-class Database(BaseObject):
+# =============================================================================
+# Databases
+# =============================================================================
+# TODO: maybe AssetModel, DatabaseType, VIPType, VirtualServerModel could have
+# common class
+class DatabaseType(NamedMixin):
+    class Meta:
```
Comment (ar4s) [inline src/ralph/data_center/models/virtual.py]: please inherit ``Meta`` from ``NamedMixin.Meta`` (see #1596)

File: src/ralph/data_center/models/virtual.py (hunk 1 above)
Comment (vi4m) [inline src/ralph/data_center/models/virtual.py]: you lost the game

Comment (vi4m) [inline src/ralph/data_center/models/virtual.py]: If it's not checked by travis/pylint/flake8:
either add missing rule
or shut up

Comment (szok) [inline src/ralph/data_center/models/virtual.py]: But this is "our" style of writing code.

Comment (vi4m) [inline src/ralph/data_center/models/virtual.py]: All of your rules should be checked by tools like pylint/flake8 or custom made scripts. Could you provide such ?

**Comment by vi4m** [comment]
ping

**Pull request closed without merging**